### ISSUE TITLE
Update Documentation | Start Here

### ISSUE THREAD

**Issue by demilolu** [Call-for-Code-for-Racial-Justice/Hacktoberfest#2]
## Background

This issue provides directions on how you can contribute to our projects by updating our documentation. Please keep this issue open.

You can make both technical and non-technical updates to documentation. 

## Tasks

- [ ] Check out the open documentation issues
- [ ] Read & follow the instructions in the Getting Started section of the readme.md to set the solution up (find projects[ here](https://call-for-code-for-racial-justice.github.io/Hacktoberfest/#/projects/legit-info)) 

If any directions are unclear, outdated, and are not addressed in [open documentation issues](https://github.com/search?q=user%3Acall-for-code-for-racial-justice+label%3Ahacktoberfest+label%3Adocumentation&type=Issues): 
- [ ] create a new issue using the documentation template
- [ ] label it 'hacktoberfest', 'documentation' assign the issue to yourself,
- [ ] Discuss your changes in the issue 

Once your documentation idea is approved

- [ ] Fork the repository.
- [ ] update the readme.md file with your changes
- [ ] Commit your changes to your fork & submit your pull request
- [ ]  Handle any feedback before the request is merged.
- [ ] PR merged by team
- [ ] Congratulations! 

If you are unfamilar with pull requests, committing code on Github, we recommend you check out our free course on open source: [ibm.biz/opensourcecourse](ibm.biz/opensourcecourse)
